journal/modernized/01-journal.adoc

```diff
@@ -1516,8 +1516,8 @@ Then he intending and preparing to come down a great steep ditch,
 I stepped down to lend him my hand to help him;
 another priest would have stepped between me and him,
 but Lord Herbert refused the priest`'s help; and stopping a little, said to the priest.
-Here is a brother that stands by will say.
-The blind leads the blind, and both will fall into the ditch.
+`'Here is a brother that stands by will say,
+The blind leads the blind, and both will fall into the ditch.`'
```

Comment:
Yeah, this is another tough one. I looked it up in the original and checked a few different editions, and they punctuated this chunk differently. I think, after reflecting, that this holds up pretty well as a direct quote, and the quote _contains_ a quote, so I think I would vote for this:

```ASCIIDOC
but Lord Herbert refused the priest`'s help; and stopping a little, said to the priest:
"`Here is a brother that stands by will say,
'`The blind leads the blind, and both will fall into the ditch.`'`"
The priest was so drunk, that he could not stand by himself.
```

Notice, I added a colon on line 1518.